.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+<!---
+Pull Request Template
+- Please fill the sections below as described in the comments.
+- This information simplifies collaboration and helps future readers.
+- Reviewers should ask for this info to be filled when missing.
+-->
+
+<!---
+PR Metadata
+- Title: Provide a short summary of the PR changes.
+- Assignees: Assign yourself and other direct collaborators.
+- Type: Create the PR as a draft. Only set it to non-draft when it is ready to be reviewed.
+- Reviewers: Only assign reviewers when the PR is ready to be reviewed.
+- Labels: Only assign labels with high priority and important context. Default is label-less.
+-->
+
+### Check the Pull Request Type:
+<!--- Put an `x` in the box that applies. -->
+
+1. [ ] Implementation of new feature(s)
+2. [ ] Bug fixes or enhancements
+
+### Description
+<!--- Describe your changes in detail -->
+
+### Related Issue
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
```

Comment:
I think is this not completely right. Some PRs are generated directly out of clickup issues.

I think we should request either of those, clickup task or github issue.